.github/workflows/ci.yml

```diff
@@ -45,9 +45,9 @@ jobs:
         with:
           python-version: "3.7.x"
           architecture: "x64"
-      - run: pip install pytest pytest-cov
+      - run: pip install --no-deps -e .
```

Comment:
what's the point of doing this and later on doing `pip install -e .[all]`? doesn't make any sense...

Reply:
This speeds up the case where documentation tests failed.

Reply:
removed